plugins/lookup/tss.py

```diff
@@ -26,6 +26,14 @@
         description: The integer ID of the secret.
         required: true
         type: int
+    _fetch_attachments:
```

Comment:
Please remove the leading underscores. They are reserved for internal or special parameters.
```suggestion
    fetch_attachments:
```

Reply:
Removed leading underscores